cla_public/apps/checker/forms.py

```diff
@@ -456,34 +521,50 @@ def title(self):
-        ),
+        description=_(u"Money you pay your landlord for rent. Do not include rent that is paid by Housing Benefit"),
         partner_description=_(
             u"Money you and your partner pay your landlord "
             u"for rent. Do not include rent that is paid by "
             u"Housing Benefit"
         ),
         choices=money_intervals_except("per_4week"),
-        validators=[MoneyIntervalAmountRequired(_(u"Enter 0 if you don’t pay rent"))],
+        validators=[
+            MoneyIntervalAmountRequired(
+                message=_(u"Tell us how much rent you pay, or enter 0 if you don’t pay rent"),
+                freq_message=_(u"Tell us how often you pay this rent"),
+                amount_message=_(
+                    u"Tell us how much rent you pay"
+                ),  # this is followed by the time period, e.g. "... each week"
+            )
+        ],
     )
     maintenance = PartnerMoneyIntervalField(
         label=_(u"Maintenance"),
         description=_(u"Money you pay to an ex-partner for their living costs"),
-        partner_description=_(u"Money you and/or your partner pay to an " u"ex-partner for their living costs"),
-        validators=[MoneyIntervalAmountRequired(_(u"Enter 0 if this doesn’t apply to you"))],
+        partner_description=_(u"Money you and/or your partner pay to an ex-partner for their living costs"),
+        validators=[
+            MoneyIntervalAmountRequired(
+                message=_(u"Tell us how much maintenance you pay, or enter 0 if this doesn’t apply to you"),
+                freq_message=_(u"Tell us how often you pay this maintenance"),
+                amount_message=_(
+                    u"Tell us how much maintenance you pay"
+                ),  # this is followed by the time period, e.g. "... each week"
+            )
+        ],
     )
     income_contribution = PartnerMoneyField(
         label=_(u"Monthly Income Contribution Order"),
-        description=_(u"Money you pay per month towards your criminal legal aid"),
-        partner_description=_(u"Money you and/or your partner pay per month " u"towards your criminal legal aid"),
-        validators=[InputRequired(_(u"Enter 0 if this doesn’t apply to you"))],
+        description=_(u"Money you pay per month towards your Criminal Legal Aid"),
+        partner_description=_(u"Money you and/or your partner pay per month towards your Criminal Legal Aid"),
+        validators=[
+            InputRequired(
+                _(u"Tell us how much you pay towards Criminal Legal Aid, or enter 0 if this doesn’t apply to you")
+            )
+        ],
     )
     childcare = PartnerMoneyIntervalField(
         label=_(u"Childcare"),
-        description=_(
-            u"Money you pay for your child to be looked after while " u"you work or study outside of your home"
-        ),
+        description=_(u"Money you pay for your child to be looked after while you work or study outside of your home"),
```

Comment:
There is a difference and it is extremely obscure - the error message text has not been okayed by Josh, so this will be picked up in LGA-940.  Until that is done we will use the default text for this.